distribution/archives/integ-test-zip/build.gradle

```diff
@@ -17,7 +19,7 @@ group = "org.elasticsearch.distribution.integ-test-zip"
 
 tasks.named("processTestResources").configure {
   inputs.properties(project(':distribution').restTestExpansions)
-  MavenFilteringHack.filter(it, project(':distribution').restTestExpansions)
+  it.filter("tokens" : project(':distribution').restTestExpansions, ReplaceTokens.class)
```

Comment:
FWIW, we can just do `filter` here instead of `it.filter`. Can we update all the other build scripts appropriately? Unnecessarily qualifying this is just kind of confusing.

Reply:
a2efdd9